xrootd/protocol/handshake/handshake.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package handshake contains the structures describing request and response
+// for handshake request (see XRootD specification).
+package handshake // import "go-hep.org/x/hep/xrootd/protocol/handshake"
+
+type ServerType int32
```

Comment:
please document this type.
it can be just some very short introductory text + a link to the XRootD specs.